ide/coqide/dune

```diff
@@ -23,19 +23,24 @@
  (files (idetop.bc as coqidetop.byte)))
 
 ; IDE Client, we may want to add the macos_prehook.ml conditionally.
+(library
+ (name coqide_config)
+ (wrapped false)
+ (modules config))
```

Comment:
I don't like having a generic name like `config` not wrapped

Reply:
Library is private tho, but we can wrap.

Reply:
Library gone.